spec/02-integration/03-dao/03-crud_spec.lua

```diff
@@ -326,10 +326,15 @@ helpers.for_each_dao(function(kong_config)
         factory:truncate_tables()
 
         for i = 1, 100 do
+          local https_only = true
+          if i == 5 then
+            https_only = false
+          end
           local api, err = apis:insert {
             name = "fixture_" .. i,
             hosts = { "fixture" .. i .. ".com" },
-            upstream_url = "http://fixture.org"
+            upstream_url = "http://fixture.org",
+            https_only = https_only
```

Comment:
can just be `https_only = i == 5` but that's fine

Reply:
Correct, we can use `i ~= 5` here.